manifests/check.pp

```diff
@@ -116,16 +116,34 @@
 
   $check_name = regsubst(regsubst($name, ' ', '_', 'G'), '[\(\)]', '', 'G')
 
-  file { "/etc/sensu/conf.d/checks/${check_name}.json":
+  case $::osfamily {
+    'windows': {
+      $etc_dir = 'C:/opt/sensu'
+      $conf_dir = "${etc_dir}/conf.d"
+      $user = undef
+      $group = undef
+      $file_mode = undef
```

Comment:
Thanks - the last version of Windows I used was 2008, and not with puppet so my knowledge is old.  Makes sense.